### PR TITLE
Transaction Outcomes endpoint

### PR DESCRIPTION
**Pull request by krzlabrdx**

### CODE REVIEW

File: src/RadixDlt.NetworkGateway.GatewayApi/Services/ITransactionOutcomeService.cs [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+public interface ITransactionOutcomeService
+{
+    Task<GatewayModel.TransactionCommittedOutcomeResponse> HandleOutcomeRequest(
+        GatewayModel.LedgerState ledgerState,
+        long stateVersion,
+        CancellationToken token = default);
+}
+
+internal class TransactionOutcomeService : ITransactionOutcomeService
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/Services/ITransactionOutcomeService.cs:87]: we should rename filename to match class not interface.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
-## 1.0.2 - Babylon
+## 1.1.0 - Babylon
 Release Date: _unreleased_
 
 - Changed default configuration value of MaxPageSize for endpoints to 100. Validate if max page size is higher than DefaultPageSize.
-- Added new optIn `receipt_output` to `/stream/transactions`, and `/transaction/committed-details` endpoints. Temporarily set by default to true, to allow client's migration.
+- Added `/transaction/outcomes` endpoint - a proxy to Node's Core API # TODO TODO document
```
Comment (PawelPawelec-RDX) [inline CHANGELOG.md:5]: you left todo here and endpoint path is also wrong, it's /transaction/committed-outcome

File: src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml [hunk 3]
```diff
@@ -282,6 +282,30 @@ paths:
                 $ref: "#/components/schemas/TransactionCommittedDetailsResponse"
         "4XX":
           $ref: "#/components/responses/ErrorResponse"
+  "/transaction/committed-outcome":
+    post:
+      operationId: TransactionCommittedOutcome
+      summary: Get Committed Transaction Outcome
+      description: |
+        Returns the committed outcome of the transaction for a given transaction identifier.
+        Transaction identifiers which don't correspond to a committed transaction will return a `TransactionNotFoundError`.
```
Comment (PawelPawelec-RDX) [inline src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml:291]: out of curiosity will it support both commited success and commited failures? hmm what would be outcome of committed failure...

**Review by PawelPawelec-RDX — CHANGES_REQUESTED** [review]

File: src/RadixDlt.NetworkGateway.GatewayApi/Services/ITransactionOutcomeService.cs (hunk 1 above)
Comment (krzlabrdx) [reply src/RadixDlt.NetworkGateway.GatewayApi/Services/ITransactionOutcomeService.cs]: Changed.

File: CHANGELOG.md (hunk 2 above)
Comment (krzlabrdx) [reply CHANGELOG.md]: Changed.

File: src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml (hunk 3 above)
Comment (krzlabrdx) [reply src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml:291]: 1. I've added committed tx info (lightweight object) to convey successful/faulted info.
2. For failed tx you might still be a subject to various fees.

**Review by PawelPawelec-RDX — APPROVED** [review]

**Pull request merged**